package.json

```diff
@@ -32,24 +32,23 @@
     "mxmlc": "./bin/mxmlc"
   },
   "scripts": {
-    "prepublish": "node prepublish.js; if [ \"$npm_package_config_npm_cmd_ran\" == \"install\" ]; then npm run-script postinstall; fi; npm config delete $npm_package_name:npm_cmd_ran",
+    "prepublish": "node prepublish.js",
```

Comment:
the existing `prepublish` script wasn't working on node `10.15.0` on Win10-64. I couldn't get this package to run `prepublish.js`, unless putting it by itself....judging by the output in the console, I'm not so sure the previous `npm config set/get/delete` logic is still needed...